src/store.js

```diff
@@ -0,0 +1,122 @@
+// @flow
```

Comment:
please create a reducers folder and put the reducers in there. you will also need to export the initial reducer from there as well